app/api/views/brief_assessors.py

```diff
@@ -0,0 +1,111 @@
+from flask import jsonify, request
+from flask_login import login_required
+
+from app.api import api
+from app.api.services import brief_assessors, users
+from app.api.helpers import role_required, is_current_user_in_brief
+from app.swagger import swag
+
+
+@api.route('/brief/<int:brief_id>/assessors', methods=["GET"], endpoint='get_brief_assessors')
+@login_required
+@role_required('buyer')
+@is_current_user_in_brief
+def get(brief_id):
+    """All brief assessors (role=buyer)
+    ---
+    tags:
+      - brief
```

Comment:
Does this need to be changed to `assessors`?

Reply:
yeah - i just thought to keep all the `/brief` routes together